tools/codesigningtool/codesigningtool.py

```diff
@@ -125,6 +125,7 @@ def _certificate_fingerprint(identity):
   ], inputstr=identity, raise_on_failure=True)
   fingerprint = fingerprint.strip()
   fingerprint = fingerprint.replace("SHA1 Fingerprint=", "")
+  fingerprint = fingerprint.replace("sha1 Fingerprint=", "")
```

Comment:
That works. I do wonder if we should go a step further and make it into a match, to remove the need for the `strip` before and `replace` after.

Thoughts @BalestraPatrick @thii @luispadron?

Reply:
Looks like we use `re.search` elsewhere in this script already ([example](https://github.com/bazelbuild/rules_apple/blob/9ec1a53129ee8f9459445daa74edd6114592f025/tools/codesigningtool/codesigningtool.py#L236)), so I would be in favour of using that since it seems like we're performing quite a few operations that could be consolidated into a single regex here.

Reply:
Either way LGTM. The string is simple and deterministic enough that just doing some replacements is fine (and is probably faster).